### PR TITLE
new metaflags added

### PR DESCRIPTION
**Pull request by uukhas**
Should solve #166.

Once this is accepted, I will merge it into #148 and then replace some bad things there.

Here, instead of just continuous appending of the new setups for variables the code looks, whether they exist in required files or not. If yes, it replaces settings, if not - appends.

### CODE REVIEW

**Comment by Expander** [comment]
That looks very nice, thanks!

Somehow it seems you and @wkotlarski are submitting the same patch, see #168.  :)
Could you both please have a look at your fellow's code? Maybe we can do a vote! :)

@uukhas : It seems you have not declared the symbols `FSLOOPTOOLS`, `FSFFLITE`, `FSCOLLIER` in `FlexibleSUSY.m`, as @wkotlarski .

Another thing: I'd propose to rename the symbols as @wkotlarski : 
~~~
FSLOOPTOOLS -> FSLoopTools
FSFFLITE -> FSFFlite
~~~

**Comment by Expander** [comment]
Many thanks! What about `FSCOLLIER`?

**Comment by uukhas** [comment]
Collier does not exist yet. This flag will be added in #148.

**Comment by Expander** [comment]
Ah, you are of course right! :)

**Comment by Expander** [comment]
I just spoke with @wkotlarski and he is fine with merging your PR.

**Comment by Expander** [comment]
Many thanks again!

**Comment by wkotlarski** [comment]
What about my question? :)

**Comment by Expander** [comment]
Which question?

**Comment by wkotlarski** [comment]
Sorry, I'm an idiot. Forgot to press a button.

File: meta/FlexibleSUSY.m [hunk 1]
```diff
@@ -353,6 +353,12 @@ FlexibleSUSY model file (FlexibleSUSY.m).
 
 FSDebugOutput = False;
 
+FSLoopLibraries::usage = "Contains a List of enabled loop libraries.";
+FSLoopLibraries = { FSSOFTSUSY };
```
[the remark this answers is not in the document]
Comment (Expander) [reply meta/FlexibleSUSY.m:359]: I think having a default is a good idea, so the setup as self-consistent as possible, even without running `configure`.

File: meta/FlexibleSUSY.m [hunk 2]
```diff
@@ -353,6 +353,12 @@ FlexibleSUSY model file (FlexibleSUSY.m).
 
 FSDebugOutput = False;
```
[the remark this answers is not in the document]
Comment (Expander) [reply meta/FlexibleSUSY.m:354]: Sorry, I thought to do this in a separate PR. But that's an excellent idea!

**Pull request merged**